datasets/mdd/README.md

```diff
@@ -0,0 +1,523 @@
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- dialogue-modeling
+---
+
+# Dataset Card for MDD
+
+## Table of Contents
+- [Dataset Card for MDD](#dataset-card-for-dataset-name)
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset 
+  Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:**[The bAbI project](https://research.fb.com/downloads/babi/)
+- **Repository:**
+- **Paper:** [arXiv Paper](https://arxiv.org/pdf/1511.06931.pdf)
+- **Leaderboard:**
+- **Point of Contact:** 
+### Dataset Summary
+
+The Movie Dialog dataset (MDD) is designed to measure how well models can perform at goal and non-goal orientated dialog centered around the topic of movies (question answering, 
+recommendation and discussion), from various movie reviews sources such as MovieLens and OMDb.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The data is present in English language as written by users on OMDb and MovieLens websites.
+
+## Dataset Structure
+
+### Data Instances
+An instance from the `task3_qarecs` config's `train` split:
+```
+{'dialogue_turns': {'speaker': [0, 1, 0, 1, 0, 1], 'utterance': ["I really like Jaws, Bottle Rocket, Saving Private Ryan, Tommy Boy, The Muppet Movie, Face/Off, and Cool Hand Luke. I'm looking for a Documentary movie.", 'Beyond the Mat', 'Who is that directed by?', 'Barry W. Blaustein', 'I like Jon Fauer movies more. Do you know anything else?', 'Cinematographer Style']}}
+```
+An instance from the `task4_reddit` config's `cand-valid` split:
+```
+{'dialogue_turns': {'speaker': [0], 'utterance': ['MORTAL KOMBAT !']}}
```

Comment:
This is indeed a dialogue worthy of reddit xD

Reply:
xD Hahaha